packages/react-storage/jest.config.js

```diff
@@ -17,5 +17,6 @@ module.exports = {
   },
   modulePathIgnorePatterns: ['<rootDir>/dist/'],
   preset: 'ts-jest',
+  setupFilesAfterEnv: ['./jest.setup.ts'],
```

Comment:
Fixes error:
```bash
  ● Test suite failed to run

    TypeError: window.URL.createObjectURL is not a function
```

Reply:
Note: this is due to sideffects from the Geo component and affects all unit test suites. Once we split Geo into a separate component then this can be limited to the geo package.